OpenUpSA/wazimap-ng-ui
# Refactored HTMLBlock code

## Description
Refactored the HTMLBlock code to handle qualitative data.

1. Qualitative data is now handled correctly
2. HTML descriptions now work
3. Code that handles metadata, title, and description was pushed to the ContentBlock parent class

## Related Issue
https://trello.com/c/E9XN4wrh/803-rich-text-descriptions-as-a-user-i-would-like-to-be-able-to-click-links-in-descriptions-and-read-them-easily-so-that-i-can-more


## How to test it locally
1. Create a qualitative variable in the backend 
2. Add an html description
3. View the rich data view ensure that it is being represented correctly.


## Checklist


### Pull Request


### Code Quality


### Testing
